docs/source/technical_tutorials/visualization/visualizing_titiler-pgstac.ipynb

```diff
@@ -0,0 +1,751 @@
+{
```

Comment:
<p><code>hash</code> is too technical - maybe call it a Unique key or <s>fingerprint</s> for this search? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/423/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Good call, thanks for the suggestion</p>